README.md

```diff
@@ -1,26 +1,153 @@
 # Vault Secrets Reloader
 
-## Description
-
 Vault Secrets Reloader can periodically check if a secret that is used in watched workloads has a new version in
 Hashicorp Vault, and if so, automatically “reloads” them by incrementing an annotation value, initiating a rollout for
 the workload’s pods. This controller is essentially a complementary to `vault-secrets-webhook`, relying on it for
 actually injecting secrets into the pods of the affected workloads.
 
 If you already use the [webhook](https://github.com/bank-vaults/vault-secrets-webhook), you are probably aware of it
 only injecting secrets when the pods are created/recreated, and until now, there were no solution within the Bank-Vaults
-ecosystem to inject secrets of these workloads in a continuous manner. Vault Secrets Reloader offers Vault Secrets
+ecosystem to inject secrets into these workloads in a continuous manner. Vault Secrets Reloader offers Vault Secrets
 Webhook users an automated solution for this problem.
 
-> \[!IMPORTANT\] This is an **early alpha version** and breaking changes are expected. As such, it is not recommended
+> [!IMPORTANT] This is an **early alpha version** and breaking changes are expected. As such, it is not recommended
 > for usage in production.
 >
 > You can support us with your feedback, bug reports, and feature requests.
```

Comment:
This will not be rendered properly
```suggestion
> [!IMPORTANT] 
> This is an **early alpha version** and breaking changes are expected. As such, it is not recommended
> for usage in production.
>
> You can support us with your feedback, bug reports, and feature requests.
```